File: Neft/Navigator.swift
import SpriteKit

extension Renderer {
    class Navigator: Renderer.BaseType {
        override init(app: GameViewController){
            super.init(app: app)
            
            // NAVIGATOR_LANGUAGE
            renderer.pushAction(OutActions.NAVIGATOR_LANGUAGE)
            renderer.pushString(NSLocale.preferredLanguages()[0])
            
            // NAVIGATOR_ONLINE
            // TODO
        }
    }
}

File: packages/neft-runtime-android/app/src/main/java/io/neft/renderer/Navigator.java
package io.neft.renderer;

import java.util.Locale;

import io.neft.App;
import io.neft.client.OutAction;

final class Navigator {
    private static final App APP = App.getInstance();
    private String language;

    static void init(Navigator navigator) {
        // NAVIGATOR_LANGUAGE
        navigator.language = Locale.getDefault().getLanguage();
        APP.getClient().pushAction(OutAction.NAVIGATOR_LANGUAGE, navigator.language);

        // NAVIGATOR_ONLINE
        // TODO
    }
}
